SobokSobok/SobokSobok/Presentation/Medicine/AddMedicineSecond/AddMedicineSecondViewController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  AddMedicineSecondViewController.swift
+//  SobokSobok
+//
+//  Created by 김승찬 on 2022/01/12.
+//
+
+import UIKit
+
+enum MedicineDayType: Int, CaseIterable {
+    case everyday = 0
+    case specificDay
+    case specificPeriod
+}
+
+final class AddMedicineSecondViewController: BaseViewController {
+    
+    // MARK: - @IBOutlets
+    
+    @IBOutlet var selectDateButtons: [UIButton]!
+    @IBOutlet var medicinePeriodButton: UIButton!
+    @IBOutlet var medicineDateButtons: [UIButton]!
+    @IBOutlet var specificDayView: UIView!
+    @IBOutlet var specificPeriodView: UIView!
+    @IBOutlet var takeMedicineDayLabel: UILabel!
+    @IBOutlet var takeMedicinePeriodLabel: UILabel!
+    
+    // MARK: View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setButtons()
+    }
+    
+    override func style() {
+        view.backgroundColor = .white
+    }
+    
+    // MARK: - Functions
+    
+    private func setButtons() {
+        medicineDateButtons.forEach {
+            $0.makeRounded(radius: 10)
+        }
+        selectDateButtons.forEach {
+            $0.makeRoundedWithBorder(radius: 10, color: Color.gray300.cgColor)
+        }
+    }
+    
+    private func updateMedicineButtonAttributes(button: UIButton, width: CGFloat, boardColor: CGColor, backgroundColor: UIColor, setTitleColor: UIColor) {
+        button.layer.borderWidth = width
+        button.layer.borderColor = boardColor
+        button.backgroundColor = backgroundColor
+        button.setTitleColor(setTitleColor, for: .normal)
+    }
+    
+    // 코드 줄일라고 만든 함수
+    // UIViewController.Type을 매개변수로 가지고 .instanceFromNib()
+    private func presentViewController(viewControllers: UIViewController.Type) {
+        let viewController = viewControllers.instanceFromNib()
+        viewController.modalPresentationStyle = .overCurrentContext
+        viewController.modalTransitionStyle = .crossDissolve
+        self.present(viewController, animated: true)
+    }
+    
+    // MARK: - @IBActions
+    
+    @IBAction func selectMedicinePeriodButton(_ sender: UIButton) {
+        let index = sender.tag
+        switch index {
+        case MedicineDayType.everyday.rawValue:
+            sender.isSelected = !sender.isSelected
```

Comment:
각 case마다 있는 이코드는 밖에서 공통으로 꺼내쓸 수 있을 것 같아요!
```suggestion
            sender.isSelected.toggle()
```